components/artwork/esd-symbol.stanza

```diff
@@ -0,0 +1,2764 @@
+#use-added-syntax(jitx)
+defpackage ocdb/esd-symbol :
+
+; Symbols: regular, large
+; Options to pass: (layer:String,show-symbol:True|False)
+; How to use:
+; create regular esd symbol on copper layer, show ESD symbol in schematic:
+;   inst <name> : ocdb/esd-symbol/regular("copper",true) 
+; create large esd symbol on silkscreen layer, show box symbol in schematic:
+;   inst <name> : ocdb/esd-symbol/large ("silkscreen",false) 
+; use place function to control position, orientation, and top/bottom layer
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/land-patterns
+  import ocdb/generic-components
+  import ocdb/symbols
+
+pcb-landpattern esd-lp (lyr:LayerSpecifier|LayerIndex, side:Side|False) :
+  if side is Side :
+    layer(Courtyard(side as Side)) = Rectangle(23.368, 20.9296, loc(-2.2098, 2.6416))

```

Comment:
Another option in Stanza:
If you have a variable (`side`) that you'd like to check its type (`side is Side`) and cast its type (`side as Side`), you can write:
```
match(side:Side) :
  layer(Courtyard(side)) = Rectangle(23.368, 20.9296, loc(-2.2098, 2.6416))
```

This is already good, and it's not necessary to change or even to follow this suggestion in the future.

Reply:
noted